test/test_other.py

```diff
@@ -8593,6 +8593,64 @@ def test_exceptions_stack_trace_and_message(self, wasm_eh):
     for check in stack_trace_checks:
       self.assertFalse(re.search(check, err), 'Expected regex "%s" to not match on:\n%s' % (check, err))
 
+    # Rethrowing exception currently loses the stack trace before the
```

Comment:
Can this be a separate test?  The existing test is already really long..

Reply:
Done